client/pages/creer-devis.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div>
+    <div class="is-fixed-top">
+      <b-notification
```

Comment:
Plutôt utiliser la fonction dédiée :                 
`                this.$buefy.notification.open({
                    message: 'Something happened correctly!',
                    type: 'is-success'
                })`